frontend/src/components/serviceaccount/Details.tsx

```diff
@@ -15,7 +16,13 @@ export default function ServiceAccountDetails() {
       extraInfo={item && [
         {
           name: 'Secrets',
-          value: item.secrets.map(({name}) => name).join(', ')
+          value: item.secrets.map(({name}, index) => <React.Fragment key={`${name}__${index}`}>
+            <Link routeName={'secret'} params={{namespace, name}}>
+              {name}
+            </Link>
+            {index !== item.secrets.length - 1 && ','}
```

Comment:
Maybe just add one per line. I think that will look better if there are more than one link.
If you wrap the result of `item.secrets.map` in a fragment, then you don't need the previous commit.